app/uploaders/place_photo_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class PlacePhotoUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Right now, you're storing all photos locally, which is okay short-term but undesirable long-term. Fog is a way to move your files to AWS, which is a much better option. I'm cool with this feature not including AWS, but be sure to delete everything currently in public/uploads before committing! And remove the files in this dir that are part of this commit :)

Reply:
cool! Deleted those place photos in the folder and will look into using fog so we can store the images to AWS. Thanks!